WordPress/Classes/ViewRelated/Site Creation/Web Address/AddressTableViewCell.swift

```diff
@@ -0,0 +1,279 @@
+        }
+        return attributedString
+    }
+
+    private static func priceAttributedString(price: String, discount: String?) -> NSAttributedString {
+        if let discount {
+            let discountAttributes: [NSAttributedString.Key: Any] = [
+                .foregroundColor: UIColor.systemGreen,
+                .font: Appearance.regularPriceFont
+            ]
+            let priceAttributes: [NSAttributedString.Key: Any] = [
+                .foregroundColor: UIColor.systemGray,
+                .font: Appearance.smallPriceFont,
+                .strikethroughStyle: NSUnderlineStyle.single.rawValue
+            ]
+            let attributedString = NSMutableAttributedString(string: discount, attributes: discountAttributes)
+            attributedString.append(NSAttributedString(string: "\n\(price)", attributes: priceAttributes))
+            return attributedString
+        } else {
+            let attributes: [NSAttributedString.Key: Any] = [
+                .foregroundColor: UIColor.systemGray,
+                .font: Appearance.regularPriceFont
+            ]
+            return NSAttributedString(string: price, attributes: attributes)
+        }
+    }
+
+    private static func stackedViews(
+        _ subviews: [UIView],
+        axis: NSLayoutConstraint.Axis,
+        alignment: UIStackView.Alignment,
+        distribution: UIStackView.Distribution,
+        spacing: CGFloat) -> UIStackView {
+        let stackView = UIStackView(arrangedSubviews: subviews)
+        stackView.axis = axis
+        stackView.alignment = alignment
+        stackView.distribution = distribution
+        stackView.spacing = spacing
+        return stackView
+    }
+
+    // MARK: - Constants
+
+    private enum Appearance {
+
+        static let contentMargins = NSDirectionalEdgeInsets(top: 16, leading: 8, bottom: 16, trailing: 16)
+
+        static let domainFont = WPStyleGuide.fontForTextStyle(.body, fontWeight: .regular)
+        static let domainTextColor = UIColor.black
+
+        static let semiboldPriceFont = WPStyleGuide.fontForTextStyle(.callout, fontWeight: .semibold)
+        static let regularPriceFont = WPStyleGuide.fontForTextStyle(.callout, fontWeight: .regular)
+        static let smallPriceFont = WPStyleGuide.fontForTextStyle(.footnote, fontWeight: .regular)
+
+        static let tagFont = WPStyleGuide.fontForTextStyle(.footnote, fontWeight: .regular)
+        static let tagTextColor: (ViewModel.Tag?) -> UIColor = { tag in
+            guard let tag else {
+                return .clear
+            }
+            switch tag {
+            case .recommended: return .systemGreen
+            case .bestAlternative: return .systemPurple
+            case .sale: return .systemOrange
+            }
+        }
+
+        static let dotViewRadius: CGFloat = 4
+        static let dotColor: (ViewModel.Tag?) -> UIColor = Appearance.tagTextColor
+    }
+}
+
+// MARK: - Old Table View Cell Design
+
+extension AddressTableViewCell {
```

Comment:
This section of the `AddressTableViewCell` class is not new. I just wanted to group all the existing code in this extension so that we can easily remove it later once the `Domain Purchasing Feature Flag` is permanently `true`.